README.md

```diff
@@ -136,7 +150,7 @@ oc new-app -f nodejsalert-ui/resources/pipeline-app-build.yml -p IMAGE_REGISTRY=
 oc new-app -f fisalert-service/src/main/resources/pipeline-app-build.yml -p IMAGE_REGISTRY=docker-registry.default.svc:5000 -p IMAGE_NAMESPACE=rh-dev -p DEV_PROJECT=rh-dev -p TEST_PROJECT=rh-test -p PROD_PROJECT=rh-prod
 
 # import aggregated-pipeline
-oc new-app -f pipelinetemplates/pipeline-aggregated-build.yml -p IMAGE_REGISTRY=docker-registry.default.svc:5000 -p IMAGE_NAMESPACE=rh-dev -p DEV_PROJECT=rh-dev -p TEST_PROJECT=rh-test -p PROD_PROJECT=rh-prod
+oc new-app -f pipelinetemplates/pipeline-aggregated-build.yml -p IMAGE_REGISTRY=docker-registry.default.svc:5000 -p IMAGE_NAMESPACE=rh-dev -p DEV_PROJECT=rh-dev -p TEST_PROJECT=rh-test -p PROD_PROJECT=rh-prod -p API_BACK_END=http://maingateway-service-rh-dev.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p END_POINT=https://fusecicd-apicast-staging.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p SANDBOX_END_POINT=https://fusecicd-apicast-staging.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p THREESCALE_URL=https://ah-3scale-ansible-admin.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p API_TOKEN=4a2a1ce5f6a7c5f6a67234d84f647f68b690e4931429d93c65e2bdf63a6a406f
 
 # import 3scale pipeline
 oc new-app -f cicd-3scale/groovy-scripts/pipeline-template.yaml -p API_BACK_END=http://maingateway-service-rh-dev.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p END_POINT=https://fusecicd-apicast-staging.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p SANDBOX_END_POINT=https://fusecicd-apicast-staging.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p THREESCALE_URL=https://ah-3scale-ansible-admin.app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -p API_TOKEN=4a2a1ce5f6a7c5f6a67234d84f647f68b690e4931429d93c65e2bdf63a6a406f
```

Comment:
I think these developer specific project names and token values could be replaced with some placeholder instead e.g. <project>     <3scale_api_token>   etc.

Reply:
change done.